packages/atomic/src/components/result-link/result-link.tsx

```diff
@@ -7,15 +7,17 @@ export interface ResultLinkProps {
   href: string;
   target: string;
   part: string;
+  title?: string;
```

Comment:
Why is `title` optional?

Reply:
In this case, having a title on the link is important because it may be shortened. In other cases though, a title isn't necessary and may actually be annoying.